tests/unit/symmetric/hierocrypt3_debug.test.ts

```
@@ -0,0 +1,219 @@
+    for (let i = 0; i < c.length; i += 2) o.push(parseInt(c.slice(i, i + 2), 16))
+    return o
+}
+
+function toHex(b: number[]): string {
+    return b.map(x => x.toString(16).padStart(2, '0')).join('')
+}
+
+function keySchedule(keyBytes: number[]): number[][] {
+    const rounds = keyBytes.length === 16 ? 6 : keyBytes.length === 24 ? 7 : 8
+    const roundKeys: number[][] = []
+    let current = [...keyBytes]
```

ESLint

[error] 'current' is never reassigned. Use 'const' instead.

(prefer-const)